.vscode/launch.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
Nice! Can you please add this to `.gitignore` and also don't forget to optimise both SVG and PNG using the tools highlighted [here](https://github.com/PaystackHQ/nigerialogos/blob/master/contributing.md).
Also there seem to be some merge conflicts with the changed files, please check that too

Reply:
Thanks @demilad I'll just do that, the PNG and SVG are both optimized.

Reply:
Hi @saminume 
This as already been done and merged. #81

Reply:
Nice! @aeadedoyin